### PR TITLE
Translation updates & Windows bug fixes

### LINKED ISSUE #238: German About screen ("Info") says "Die Internet-Seite von GIMP öffnen"
It links to the Glimpse website, though.

(Glimpse 0.1.0, Linux, Flatpak, on Mint 19.2)

### PR DESCRIPTION
**Pull request by TrechNex**
Fixes #238 #250 #230 #240 #247 .

Changes proposed in this pull request:
- Fix various translations throughout the application to use the new naming convention
- Update AUTHORS list to include Cynthia Revström (thank you for code signing!) and skyejonke. Also set chaomodus as a maintainer.
- Update the Copyright year
- Ensure Glimpse Image Editor no longer shares the same Windows task bar icon as the GNU Image Manipulation Program when they're both running at the same time
- Merged changes from dev-g210

### CODE REVIEW

**Comment by TrechNex** [comment]
Also documented the process for fixing translations so it's not just me doing this in future 😅 

https://wiki.glimpse-editor.org/index.php?title=Building_Glimpse#Fixing_Translations

**Comment by TrechNex** [comment]
I am deliberately taking care with my commit history so we don't have to squash this.

File: AUTHORS [hunk 1]
```diff
@@ -29,13 +31,3 @@ The following people have helped to document Glimpse:
 
  chaomodus
  melody
-
-
```
Comment (TrechNex) [inline AUTHORS]: The rationale behind this change is a) we're not on Patreon anymore, b) the "Sponsor Contributor" thing was only intended to be a temporary promotion to get the Glimpse project off the ground.

Their names will still be immortalised with all copies of the v0.1.0 code, the release notes we published, and the NEWS page that will be included with all future releases thanking them for making our very first release happen.

File: AUTHORS [hunk 2]
```diff
@@ -29,13 +31,3 @@ The following people have helped to document Glimpse:
...
-The following people sponsored Glimpse on Patreon:
-
- Dominic Watson
- Kretz
- Loren Dias
- Massimo D'Ambrogio
- Roberts-Loux Foundation
- Sami Mannila
```
Comment (TrechNex) [inline AUTHORS]: I have moved this list of "sponsor contributors" to the NEWS file where they can continue to be immortalised for making our very first release happen.

The difference is that this does not imply that we have to continue maintaining that backer perk for new donors, or that we have to implement a new tab for it in the credits section of the front-end UI.

Our original intent behind "sponsor contributors" was to help us get the project off the ground and cover costs when we were still on Patreon. We are now on Open Collective, and we still thank our backers on social media and in release notes/announcements.

**Comment by TrechNex** [comment]
I'm happy with what I have here for now. I'm going to work on #240 and #250 in separate PRs.
I don't think this *completely* resolves our translation problems, but it does fix a good many of them for the most commonly used languages.

**Comment by bochecha** [comment]
Since you requested my review: that's a lot of changes all at once, not all related to each other, making it much harder to review than it should be.

I don't think I'll have the time to review something big like this any time soon, sorry.

**Comment by TrechNex** [comment]
@bochecha my apologies, you had been added to this PR by mistake. For the other people I added, I have provided guidance on the Matrix contributors channel to focus on the notable features only.

I used this PR to merge in dev branch, update documentation for a new release, apply minor bug fixes and change the proper nouns in dozens a load of translation files. I stopped making changes so this could be a "utility" PR and remaining changes could be one feature = one PR.

**Pull request closed without merging**